edn/src/types.rs

```diff
@@ -54,7 +55,19 @@ impl Display for Value {
             Boolean(v) => write!(f, "{}", v),
             Integer(v) => write!(f, "{}", v),
             BigInteger(ref v) => write!(f, "{}N", v),
-            Float(OrderedFloat(v)) => write!(f, "{}", v),
+            Float(ref v) => {
```

Comment:
[Matching on constants have been deprecated](https://github.com/rust-lang/rfcs/pull/1445), is there a more idiomatic way than if/elses?